docs/references/backend/overview.mdx

```diff
@@ -224,17 +224,17 @@ To access a resource, you must first instantiate a `clerkClient` instance.
       <Tab>
         <CodeBlockTabs options={["ESM", "CJS"]}>
           ```js
-          import { createClerkClient } from '@clerk/clerk-sdk-node'
+          import { createClerkClient } from '@clerk/express'
 
-          const clerkClient = createClerkClient({ secretKey: '{{secret}}' })
+          const clerkClient = createClerkClient({ secretKey: process.env.CLERK_SECRET_KEY })
 
           const userList = await clerkClient.users.getUserList()
           ```
 
           ```js
-          const Clerk = require('@clerk/clerk-sdk-node/cjs/instance').default
+          const Clerk = require('@clerk/express')
 
-          const clerkClient = Clerk({ secretKey: '{{secret}}' })
+          const clerkClient = Clerk.createClient({ secretKey: process.env.CLERK_SECRET_KEY })
```

Comment:
is it `createClient` or `createClerkClient`

Reply:
Updated!

Reply:
Should be `createClerkClient`, updated!